tests/VocabularyManager.UnitTests/Services/OxfordDictionaryServiceTests.cs

```diff
@@ -0,0 +1,60 @@
+using VocabularyManager.UseCases.DTOs;
+using VocabularyManager.UseCases.Exceptions;
+using VocabularyManager.UseCases.Services;
+using VocabularyManager.UseCases.Validators;
+
+namespace VocabularyManager.UnitTests.Services
+{
+    public class OxfordDictionaryServiceTests
+    {
+        [Fact]
+        public async Task Get_Word_List_By_Link_Async_Test_1()
+        {
+            //Arrange
+            var parsingService = new OxfordDictionaryParser();
+
+            //Act
+            var wordList = parsingService
+                .GetWordListByLinkAsync(
+                "https://www.oxfordlearnersdictionaries.com/wordlists/oxford3000-5000",
+                new OxfordParsingWordDTOValidator()
+            ); //well-fitable url
+
+            //Assert
+            Assert.NotNull(wordList);
+            Assert.IsAssignableFrom<IAsyncEnumerable<WordDTO>>(wordList);
+        }
+
+        [Fact]
+        public void Get_Word_List_By_Link_Async__Exceptions_Test()
```

Comment:
one test. One assert (one result assert. It's fine to check one object with multiple assertions, depending on the test)
if logic of test does not change, only input. Read about `Theory` which can be used instead of `Fact`